site/docs/v1/tech/installation-guide/_shared-state.adoc

```diff
@@ -0,0 +1,5 @@
+FusionAuth stores almost all state in its database. For clustered environments, the following interactions occur between nodes:
```

Comment:
Suggestion:

> interactions may occur between nodes

We could not that nothing breaks if we don't have this inter node communication.